playbooks/installation/index.adoc

```diff
@@ -122,15 +122,23 @@ More information on Pods & Services can be found in the link:https://docs.opensh
 
 === DNS
 
-TODO: Blurb about
+All of the hosts in the cluster need to be resolveable via DNS.  In order to test that this is working execute the following from any host in your cluster to each other host in your cluster.   Additionally if using a control node it too should be able to resolve all hosts in your cluster. Running the following with the targeted host names and the IP address of you DNS server should confirm resolution. 
+
+`$ dig <node_hostname> @<IP_address> +short`
```

Comment:
So, I think in this section, should simply describe the problem space and the solution. Instructing the user to run commands when no infrastructure exists yet could be confusing.